### PR TITLE
TLM-657 Splunk UI enhancement 

### PR DESCRIPTION
**Pull request by Nikii1118**
I have tested with Kafka:
Screenshots
splunkui working
Its updating the configuration currently only one configuration can be set.

### CODE REVIEW

**Review by garasankara — APPROVED** [review]

File: cmd/configui/configui.go [hunk 1]
```diff
@@ -54,13 +53,39 @@ type MyHec struct {
 	Index string `json:"index"`
 }
 
+func getSplunkHttpConfig(c *gin.Context, s *SystemHandler) {
+	var SplunkConfig MyHec
+	configValues, err := s.ConfigBus.Get("splunkURL")
+	fmt.Println(configValues.Value)
```
Comment (windsparks33) [inline cmd/configui/configui.go:59]: debug code should be removed.

Comment (Nikii1118) [reply cmd/configui/configui.go]: yes I removed it.

**Review by garasankara — APPROVED** [review]

File: cmd/dbdiscauth/dbdiscauth.go [hunk 2]
```diff
@@ -82,6 +101,27 @@ func addServiceToDB(db *sql.DB, service auth.Service, authService *auth.Authoriz
 	return nil
 }
 
+// splunk configuration are getting added in the database
+//TO DO: Update it in the db
+func splunkAddHECToDB(db *sql.DB, SplunkConfig auth.SplunkConfig, authService *auth.AuthorizationService) error {
+	log.Print("inside splunkaddhectodb")
```
Comment (windsparks33) [inline cmd/dbdiscauth/dbdiscauth.go:107]: Remove debug code.

File: cmd/dbdiscauth/dbdiscauth.go [hunk 3]
```diff
@@ -82,6 +101,27 @@ func addServiceToDB(db *sql.DB, service auth.Service, authService *auth.Authoriz
...
+	stmt, err := db.Prepare("INSERT INTO HttpEventCollector(url, `key`, `index`) VALUES(?, ?, ?)")
+	if err != nil {
+		return err
+	}
+	fmt.Println("reached inside splunkAddHECTODB")
```
Comment (windsparks33) [inline cmd/dbdiscauth/dbdiscauth.go:112]: Remove debug code.

File: cmd/dbdiscauth/dbdiscauth.go [hunk 4]
```diff
@@ -154,6 +194,20 @@ func initMySQLDatabase() (*sql.DB, error) {
 		}
 	}
 
+	for {
+		_, err = db.Query("CREATE TABLE IF NOT EXISTS HttpEventCollector(url VARCHAR(255), `key` VARCHAR(4096), `index` VARCHAR(4096));")
+		fmt.Println("inside second dbquery")
```
Comment (windsparks33) [inline cmd/dbdiscauth/dbdiscauth.go:199]: please remove debug code..

**Review by garasankara — APPROVED** [review]

**Review by windsparks33 — APPROVED** [review]

**Pull request merged**